database/database.js

```diff
@@ -1,22 +1,58 @@
 const mongoose = require('mongoose');
 
+// Actually connecting to the database
 mongoose.connect('mongodb://localhost:27017/sidebar?gssapiServiceName=mongodb', {useNewUrlParser: true, useUnifiedTopology: true})
```

Comment:
If you end up moving to a remote database, remember to put your urls and keys into a separate config.js file

Reply:
Good point. Yeah, that's exactly the sort of thing I usually forget.